src/main/java/com/mavericsystems/userservice/service/UserService.java

```diff
@@ -1,16 +1,19 @@
 package com.mavericsystems.userservice.service;
 
 import com.mavericsystems.userservice.dto.UserDto;
+import com.mavericsystems.userservice.dto.UserRequest;
 import com.mavericsystems.userservice.model.User;
-import org.springframework.stereotype.Service;
+
 
 import java.util.List;
 
 
 public interface UserService {
 
-    UserDto createUser(User user);
-
-    List<UserDto> getUsers();
-
+    UserDto createUser(UserRequest userRequest);
```

Comment:
Do you want to specify as the UserDto as a naming convention ? can you rename this by avoiding Dto